MODULE.bazel

```diff
@@ -4,12 +4,11 @@ module(
     repo_name = "io_grpc_grpc_proto",
 )
 
+bazel_dep(name = "googleapis", version = "0.0.0-20240326-1c8d509c5", repo_name = "com_google_googleapis")
 bazel_dep(name = "rules_proto", version = "4.0.0")
 bazel_dep(name = "protobuf", version = "23.1", repo_name = "com_google_protobuf")
 # -- bazel_dep definitions -- #
 
-http_archives_extension = use_extension("//bazel:http_archives_extension.bzl", "http_archives_extension")
-use_repo(http_archives_extension, "com_google_googleapis")
-
-switched_rules_extension = use_extension("//bazel:switched_rules_extension.bzl", "switched_rules_extension")
-use_repo(switched_rules_extension, "com_google_googleapis_imports")
+switched_rules = use_extension("@com_google_googleapis//:extensions.bzl", "switched_rules")
+switched_rules.use_languages()
+use_repo(switched_rules, "com_google_googleapis_imports")
```

Comment:
looks like we don't even need this? I don't see this referenced anywhere in this repo. does the side effect matter? I don't think it does since the only place com_google_googleapis is used is for a raw proto library, which i think always exists? https://github.com/grpc/grpc-proto/blob/d4056f5f413d38c6e05e1f14f4d208447c0e1673/BUILD.bazel#L244

Reply:
Yeah, I think you are right. Looking at when it was introduced, it seems it was already not used:
https://github.com/grpc/grpc-proto/commit/e53e1f363c14f30c8ddf42e84ce284da6f6ebff3

I think we'll just use the "if it compiles" approach for confirming this is okay. (The author of that change is around, but it was 4 years ago)